### PR TITLE
Refactor `test_detect_injected_correlation`

### PR DESCRIPTION
**Pull request by h-mayorquin**
@alejoe91 mentioned that this is a test that constantly fails. I added some determinism to the tests. Plus, I think it is easier to test for the location of the peak (which should match the delta of the injected noise) instead of the maximum of the peak (tbh, I had a hard time interpreting why should the peak_value should be around `np.median(cc_01) * 0.7 + num_injected_spikes`).

What do you think? 
I

### CODE REVIEW

File: spikeinterface/postprocessing/tests/test_correlograms.py [hunk 1]
```diff
@@ -191,41 +190,38 @@ def test_detect_injected_correlation():
...
+        peak_location_02_ms = bins[np.argmax(cc_10)]
         
-        half_bin_ms = np.mean(np.diff(bins)) / 2.
-        
-        # check that we have a peak in the CC 01
-        expected_peak_value = np.median(cc_01) * 0.7 + num_injected
-        assert np.max(cc_01) > expected_peak_value
-        assert np.max(cc_10) > expected_peak_value
+        sampling_period_ms = (1000.0 / sampling_frequency)
+        assert abs(peak_location_01_ms) - 1.4 < sampling_period_ms
```
Comment (samuelgarcia) [inline spikeinterface/postprocessing/tests/test_correlograms.py:200]: Maybe even this
```suggestion
        assert abs(peak_location_01_ms) - injected_delta_ms< sampling_period_ms
```

**Comment by samuelgarcia** [comment]
`np.median(cc_01) * 0.7 + num_injected_spikes` is bit kriptic.
The idea is have more than the basal coorelation minus a factor + injected (which is known).
Indeed the new test is really easier to read even if it is a bit diferrents.

**Comment by h-mayorquin** [comment]
Thanks for explaining @samuelgarcia. I guess it is the factor the one that I don't know where it is coming from.

Are you fine with having the new test instead?

**Comment by samuelgarcia** [comment]
The factor was very empiric.
Lets go with this!

**Review by samuelgarcia — APPROVED** [review]

**Pull request merged**